ddtrace/opentelemetry/tracer.go

```diff
@@ -17,15 +19,44 @@ import (
 var _ oteltrace.Tracer = (*oteltracer)(nil)
 
 type oteltracer struct {
-	cfg      oteltrace.TracerConfig
 	provider *TracerProvider
 	ddtrace.Tracer
 }
+type otelCtxToDDCtx struct {
+	oc oteltrace.SpanContext
+}
+
+func (c *otelCtxToDDCtx) TraceID() uint64 {
+	allTraceID := [16]byte(c.oc.TraceID())
+
+	return binary.BigEndian.Uint64(allTraceID[8:])
+}
+
+func (c *otelCtxToDDCtx) SpanID() uint64 {
+	sid := [8]byte(c.oc.SpanID())
+	return binary.BigEndian.Uint64(sid[:])
+}
+
+func (c *otelCtxToDDCtx) ForeachBaggageItem(handler func(k, v string) bool) {
+	return
+}
```

Comment:
```suggestion
func (c *otelCtxToDDCtx) ForeachBaggageItem(handler func(k, v string) bool) {}
```
(no return necessary)